plugins/admin_console/kaltura_application_links/admin/SelfserveAdminAction.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+class SelfserveAdminAction extends KalturaApplicationPlugin
+{
+    public function __construct()
+    {
+        $this->rootLabel = "External apps";
```

Comment:
the root label is not something related to all external apps?
why it should be defined in the action explicitly? if someone will add additional app action we will have multiple tabs with the same root name?